userContent.css

```diff
@@ -678,6 +678,7 @@
       --color-moz-inverse-bg: var(red) !important;
       --focus-shadow: 0 0 0 4px color-mix(in srgb, var(--in-content-primary-button-background) 30%, transparent),
         0 0 0 2px var(--in-content-primary-button-background-active);
+      scrollbar-color: var(--in-content-deemphasized-text) var(--in-content-page-background);
```

Comment:
In the past, scroll colors were specified, but the behavior was different for each OS, and some accessibility issues could occur.